### ISSUE TITLE
PMID:36650056

### ISSUE THREAD

**Issue by manulera** [pombase/fypo#4258]
* tuclear congression without nuclear fusion
* twin horsetail nucleus (consequence of)

**Comment by manulera** [comment]
It would be nice to express the fact that nuclear congression needs to happen for this to occur.

**Comment by manulera** [comment]
Also add the causality between nuclear congression without nuclear fusion leading to twin horsetail.

**Comment by ValWood** [comment]
We should double-check that these connections look sensible in the query builder and in Canto. There are 2 relations is_output_of, and the other one which I can't remember....and I am never sure how the annotations are propagated over these. This will be a good example to check.

**Comment by manulera** [comment]
Yes! I have not added them yet, just added the terms to make the annotations.
